modules/storage-page-memory/src/test/java/org/apache/ignite/internal/storage/pagememory/mv/AbstractPageMemoryMvPartitionStorageTest.java

```diff
@@ -35,8 +35,7 @@
 /**
  * Base test for MV partition storages based on PageMemory.
  */
-@ExtendWith(ConfigurationExtension.class)
-@ExtendWith(WorkDirectoryExtension.class)
+@ExtendWith({ConfigurationExtension.class, WorkDirectoryExtension.class})
```

Comment:
So they made it both `@Repeatable` and array-based. I blame junit!

Reply:
Btw, what's preferable?

Reply:
I like array more, but I did not know about it